maps/random_ruins/exoplanets/moghes/moghes_dead_guwandi.dm

```diff
@@ -0,0 +1,65 @@
+/datum/map_template/ruin/exoplanet/moghes_dead_guwandi
+	name = "Dead Guwandi"
+	id = "moghes_dead_guwandi"
+	description = "A Guwandi warrior, who found an honorable death"
+
+	template_flags = TEMPLATE_FLAG_NO_RUINS|TEMPLATE_FLAG_RUIN_STARTS_DISALLOWED
+	sectors = list(SECTOR_UUEOAESA)
+	prefix = "moghes/"
+	suffixes = list("moghes_dead_guwandi.dmm")
+	unit_test_groups = list(2)
+
+/obj/effect/landmark/corpse/moghes_dead_guwandi
+	name = "Guwandi"
+	corpseuniform = /obj/item/clothing/under/unathi/zazali
+	corpsesuit = /obj/item/clothing/suit/unathi/robe/kilt
+	corpseshoes = /obj/item/clothing/shoes/sandals/caligae
+	corpsehelmet = /obj/item/clothing/accessory/sinta_hood
+	corpsebelt = /obj/item/material/sword/longsword
+
+/obj/effect/landmark/corpse/moghes_dead_guwandi/do_extra_customization(mob/living/carbon/human/M)
+	M.ChangeToHusk()
+	M.adjustBruteLoss(rand(200,400))
+	M.dir = pick(GLOB.cardinal)
+	if(M?.w_uniform)
+		M.w_uniform.color = "#181a19"
+	if(M?.wear_suit)
+		M.wear_suit.color = "#d4d3ab"
+	if(M?.head)
+		M.head.color = "#d4d3ab"
+
+
+/obj/item/paper/fluff/guwandi
+	name = "final words"
+	desc = "A scrap of paper, with a few words jotted down on it."
+	info = "A drum with no head,<BR>\
+	A blade with no hilt,<BR>\
+	A song with no voice,<BR>\
+	Still, it is mine."
+
+/obj/item/paper/fluff/guwandi/Initialize()
+	. = ..()
+	var/languagetext = "\[lang=p\]"
+	languagetext += "[info]\[/lang\]"
+	info = parsepencode(languagetext)
+	icon_state = "paper_words"
```

Comment:
should be using stuff added in <https://github.com/Aurorastation/Aurora.3/pull/18850>